specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServersTest/preview/2024-12-01-preview/examples/AdvancedThreatProtectionSettingsGet.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
All the changes within `mysql` folder will be reverted as it's only for testing purposes.